function-client/src/main/java/io/micronaut/function/client/http/HttpFunctionExecutor.java

```diff
@@ -65,26 +64,23 @@
      * @param threadFactory threadFactory
      * @param nettyClientSslBuilder nettyClientSslBuilder
      * @param codecRegistry codecRegistry
-     * @param annotationMetadataResolver annotationMetadataResolver
-     * @param filters filters
+     * @param filterResolver filter resolver
      */
     public HttpFunctionExecutor(
             HttpClientConfiguration configuration,
             @Named(NettyThreadFactory.NAME) ThreadFactory threadFactory,
             NettyClientSslBuilder nettyClientSslBuilder,
             MediaTypeCodecRegistry codecRegistry,
-            AnnotationMetadataResolver annotationMetadataResolver,
-            HttpClientFilter... filters) {
+            FilterResolver filterResolver) {
```

Comment:
This is a breaking change

Reply:
Looks you are right. Before I spend time on fixing it can @jameskleeh or @graemerocher please confirm if it was made public available intentionally? I was not been able to find any documentation on how clients should use this class and a quick search on github does not show opensource projects actually use it directly